common/test/acceptance/pages/studio/library.py

```diff
@@ -168,6 +169,7 @@ def count(self, count):
         Sets value of children count input
         """
         count_text = self.get_setting_element(self.COUNT_LABEL)
+        count_text.send_keys(Keys.BACK_SPACE)
```

Comment:
You're right Jesse but In our scenario It always will be 1 when we open fresh Edit pop up. If you want me to implement that "select all values" and then delete It can be done.